Gemfile

```diff
@@ -1,2 +1,4 @@
 source 'https://rubygems.org'
 gemspec
+
+gem 'rails'
```

Comment:
This is the default `Gemfile` for `script/server` (replaces the old `test/Gemfile`).